server/src/main/java/com/distkv/server/Node.java

```diff
@@ -0,0 +1,68 @@
+package com.distkv.server;
+
+import java.io.Serializable;
+
+/**
+ * the node information
+ */
+public class Node implements Serializable {
+
+  private static final long serialVersionUID = 1210798805637667776L;
+
+  private String nodeId;
```

Comment:
NodeId is defined https://github.com/distkv-project/distkv/blob/master/common/src/main/java/com/distkv/common/id/NodeId.java 

Please use this instead of a string.

Reply:
yes, and the nodeId should be determined by meta server.